services/traction/api/db/models/v1/presentation_requests.py

```diff
@@ -0,0 +1,125 @@
+"""Verification requests for proof.
+
+Models of the Traction tables for Connection Invitation and related data.
+
+"""
+import uuid
+from typing import List, Optional
+from sqlalchemy.orm import selectinload
+from sqlmodel import Field, Relationship, desc
+from sqlalchemy import Column, text, String, select
+from sqlalchemy.dialects.postgresql import UUID, JSON, ARRAY
+from sqlmodel.ext.asyncio.session import AsyncSession
+
+from api.db.models.base import StatefulModel, TimestampModel
+
+from api.db.models.v1.contact import Contact
+
+from api.endpoints.models.v1.errors import (
+    NotFoundError,
+)
+
+
+class VerifierPresentationRequest(StatefulModel, TimestampModel, table=True):
+    """Verifier Presentation Request
+
+    Model for the Verifier Presentation Request table (postgresql specific
+    dialects in use).This will track verifications of proof for the Tenants.
+
+    Attributes:
+    """
+
+    __tablename__ = "verifier_presentation_request"
+
+    v_presentation_request_id: uuid.UUID = Field(
+        sa_column=Column(
+            UUID(as_uuid=True),
+            primary_key=True,
+            server_default=text("gen_random_uuid()"),
+        )
+    )
+
+    tenant_id: uuid.UUID = Field(
+        foreign_key="tenant.id", index=True
+    )  # TODO why isn't this in base
+    contact_id: uuid.UUID = Field(foreign_key="contact.contact_id", index=True)
+    external_reference_id: str = Field(nullable=True)
```

Comment:
that's built into the proof_request object and would end up in the `proof_request` attribute. I don't think we want to duplicate it at the top level.

Reply:
well, that is true, it certainly makes it hard for someone looking at their list to differentiate items. we have duplication of data that resides in the related acapy/ledger objects all over the place. version may be of little use, but name could be very useful (we will have to see if anyone actually uses it.  when the presentation request is made using a template it would be fairly simple to identify what this presentation was for, but in an ad-hoc manner, you have no ability to correlate without asking for the acapy object and looking at it, seems like a friction point.